src/typescript.rs

```diff
@@ -29,26 +29,28 @@ pub fn convert_type(ty: &syn::Type) -> TsType {
             let arguments = &segment.arguments;
             let identifier = ident.to_string();
             match identifier.as_str() {
-                "i8" => "number".to_string().into(),
-                "u8" => "number".to_string().into(),
-                "i16" => "number".to_string().into(),
-                "u16" => "number".to_string().into(),
-                "i32" => "number".to_string().into(),
-                "u32" => "number".to_string().into(),
-                "i64" => "number".to_string().into(),
-                "u64" => "number".to_string().into(),
-                "i128" => "number".to_string().into(),
-                "u128" => "number".to_string().into(),
-                "isize" => "number".to_string().into(),
-                "usize" => "number".to_string().into(),
-                "f32" => "number".to_string().into(),
-                "f64" => "number".to_string().into(),
-                "bool" => "boolean".to_string().into(),
-                "char" => "string".to_string().into(),
-                "str" => "string".to_string().into(),
-                "String" => "string".to_string().into(),
-                "NaiveDateTime" => "Date".to_string().into(),
-                "DateTime" => "Date".to_string().into(),
+                "i8" => "number".to_owned().into(),
+                "u8" => "number".to_owned().into(),
+                "i16" => "number".to_owned().into(),
+                "u16" => "number".to_owned().into(),
+                "i32" => "number".to_owned().into(),
+                "u32" => "number".to_owned().into(),
+                "i64" => "number".to_owned().into(),
+                "u64" => "number".to_owned().into(),
+                "i128" => "number".to_owned().into(),
+                "u128" => "number".to_owned().into(),
+                "isize" => "number".to_owned().into(),
+                "usize" => "number".to_owned().into(),
+                "f32" => "number".to_owned().into(),
+                "f64" => "number".to_owned().into(),
+                "bool" => "boolean".to_owned().into(),
+                "char" => "string".to_owned().into(),
+                "str" => "string".to_owned().into(),
+                "String" => "string".to_owned().into(),
+                "NaiveDateTime" => "Date".to_owned().into(),
+                "DateTime" => "Date".to_owned().into(),
+                "Uuid" => "string".to_owned().into(),
```

Comment:
🙌

Reply:
(thanks for adding Uuid here)

Reply:
No problem. I actually needed it in my use case, so it worked out.